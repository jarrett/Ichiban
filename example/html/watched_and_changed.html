<p>This is the new text.</p>