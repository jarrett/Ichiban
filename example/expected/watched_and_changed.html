<!DOCTYPE html>
<html>
  <head>
    <title>Example Site</title>
  </head>
  
  <body>
    <h1>Example Site</h1>
    
    <p>This is the new text.</p>
  </body>
</html>